views/job-role.html

```diff
@@ -0,0 +1,46 @@
+{% extends "_includes/layouts/base.html" %}
+{% block content %}
+<main class="container mt-4">
+       
+    <section id="orders" class="">
+
+        <h1>{{ jobRole[0].jobRoleTitle }}</h1>
+        {% if jobRole and jobRole.length > 0 %}
+
+        {% for role in jobRole %}
+        
+        <div class="specDiv">
+            <h4>Job Specification</h4>
+
+            <div class="specListDiv">
+                <ul>
+                        {% if role.specificationText %}
+                            {% set specArray = role.specificationText.split(', ') %}
+                            {% for specText in specArray %}
+                                <li>{{ specText }}</li>
+                            {% endfor %}
+                        {% else %}
+                            <li>No specification available</li>
+                        {% endif %}
+                    </ul>
+            </div>
+
+        </div>
+        
+        <a href="{{ role.sharepointLink }}">Sharepoint Link</a>
```

Comment:
Can you add a `<br>` after this so the`SharePoint` and `Back` links aren't on the same line